tests/e2e/e2e_test.go

```diff
@@ -45,9 +45,9 @@ var (
 // TODO: Find more scalable way to do this
 func (s *IntegrationTestSuite) TestAllE2E() {
 	// Reset the default taker fee to 0.15%, so we can actually run tests with it activated
-	s.T().Run("SetDefaultTakerFeeBothChains", func(t *testing.T) {
-		s.T().Log("resetting the default taker fee to 0.15%")
-		s.SetDefaultTakerFeeBothChains()
+	s.T().Run("SetDefaultTakerFeeChainB", func(t *testing.T) {
+		s.T().Log("resetting the default taker fee to 0.15% on chain B only")
+		s.SetDefaultTakerFeeChainB()
```

Comment:
Is it not possible for us to just modify the genesis of the chain to do this? Feels excessive to do this as txs.